src/modules/client/client.service.ts

```diff
@@ -0,0 +1,153 @@
+import { Skill } from './../skill/entities/skill.entity';
+import { Freelancer } from 'src/modules/freelancer/entities/freelancer.entity';
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Any, ArrayOverlap, In, Repository } from 'typeorm';
+import { Users } from '../entities/users.entity';
```

Comment:
Absolute path here as well, I'd suggest you check if your editor is configured to not import the relative path

Reply:
Okay, I will check it, thanks